app/reports/forms.py

```diff
@@ -47,16 +47,13 @@ class IncidentReportForm(Form):
     longitude = HiddenField('Longitude')
     location = StringField('Address')
 
-    date = DateField('Date', default=datetime.date.today(),
+    date = DateField('Date (year-month-day)', default=datetime.date.today(),
```

Comment:
Since someone will want to mark that an incident happened at 3:04 PM on March, 3 2015